standalone.js

```diff
@@ -8,40 +8,35 @@ const getSupportInfo = require("./src/main/support").getSupportInfo;
 const internalPlugins = [
   require("./src/language-js"),
   require("./src/language-css"),
-  require("./src/language-handlebars"),
+  // require("./src/language-handlebars"),
   require("./src/language-graphql"),
   require("./src/language-markdown"),
-  require("./src/language-html"),
+  // require("./src/language-html"),
```

Comment:
I'd rather remove the lines than keep commented code. We can add them once we're ready for them